### PR TITLE
Finished: Find minimum exercise

### LINKED ISSUE #215: Find  Minimum
[Eloquent JavaScript: Minimum](http://eloquentjavascript.net/03_functions.html#h_XTmO7z7MPq)

### PR DESCRIPTION
**Pull request by DeuceX**
Closed #215

### CODE REVIEW

File: 01-syntax/02-functions-variables-scope-and-memory/min.js [hunk 1]
```diff
@@ -6,6 +6,13 @@
 
 // → Your code here.
```
Comment (nesterone) [inline 01-syntax/02-functions-variables-scope-and-memory/min.js]: Please, remove unnecessary  comment

**Comment by DeuceX** [comment]
Unnecessary comment removed

File: 01-syntax/02-functions-variables-scope-and-memory/min.js [hunk 2]
```diff
@@ -4,7 +4,12 @@
...
 
-// → Your code here.
+function min(a, b) {
+  if (a > b) {
+    return b;
+  }
+  return a;
+}
 
 console.log(min(0, 10));
```
Comment (nesterone) [inline 01-syntax/02-functions-variables-scope-and-memory/min.js:17]: Please, provide more test cases

File: 01-syntax/02-functions-variables-scope-and-memory/min.js [hunk 3]
```diff
@@ -4,9 +4,25 @@
  */
 /* global min */
 
-// → Your code here.
+function min(a, b) {
+  if (isNaN(a) || isNaN(b)) {
+    return 'Error! Wrong input';
```
Comment (nesterone) [inline 01-syntax/02-functions-variables-scope-and-memory/min.js]: replace with https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/throw

**Pull request merged**